### PR TITLE
seo enhancement

### PR DESCRIPTION
**Pull request by nickhsine**
@YuCJ

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -58,13 +58,41 @@ app.use(function (req, res, next) {
...
+    }
+  })
+})
 
 app.get('/robots.txt', (req, res) => {
   res.format({
     'text/plain': function () {
-      res.status(200).render('robots')
+      res.status(200).send('User-agent: * \n' +
+      'Sitemap: https://www.twreporter.org/sitemaps/twreporter-sitemap.xml')
```
Comment (nickhsine) [inline src/server.js]: if you want to know the details, you can see https://staging.twreporter.org/sitemaps/twreporter-sitemap.xml first.

File: src/server.js [hunk 2]
```diff
@@ -58,13 +58,43 @@ app.use(function (req, res, next) {
...
 })
 
-// Add max-age: 900 on response header of /sw.js
-app.get('/sw.js', Express.static(path.join(__dirname, '..'), { maxAge: 900 }))
+app.get('/sw.js', function (req, res, next) {
+  const options = {
+    headers:{
+      'Cache-Control': 'public, max-age=900, must-revalidate'
+    },
+    root: path.resolve(__dirname + '/../')
```
Comment (YuCJ) [inline src/server.js]: `/../` will always resolve to root `/`

File: src/server.js [hunk 3]
```diff
@@ -58,13 +58,43 @@ app.use(function (req, res, next) {
...
+    }
+  })
+})
+
+app.get('/BingSiteAuth.xml', function (req, res, next) {
+  const options = {
+    headers: {
+      'Cache-Control': 'public, max-age=2419200, must-revalidate'
+    },
+    root: path.resolve(__dirname + '/../static/')
```
Comment (YuCJ) [inline src/server.js]: `/../static/` will always resolve to `/static`

**Comment by nickhsine** [comment]
Review patched. please help to verify.

**Comment by YuCJ** [comment]
nice 👍

**Pull request merged**